test/engine_libkbfs.go

```diff
@@ -52,6 +52,9 @@ func (k *LibKBFS) InitTest(ver libkbfs.MetadataVer,
 	config.SetMetadataVersion(ver)
 
 	setBlockSizes(k.tb, config, blockSize, blockChangeSize)
+	if batchSize > 0 {
+		config.SetBGFlushDirOpBatchSize(batchSize)
+	}
```

Comment:
Background flushing is disabled entirely by default in these DSL tests, so the flush period has no effect.  We do that to make it easier to have more control about when things flush.

Really the only reason to set this parameter is to go into `non-admin` mode and force a `SyncAll` during every directory operation, rather than waiting until the end of the `as` block.